### ISSUE TITLE
A user with a public or private FOAF should be able to connect with it

### ISSUE THREAD

**Issue by jmvanel** [read-write-web/rww-play#99]
Somehow a user with a public or private FOAF profile should be able to subscribe without entering her/his mail ( because the  FOAF profile contains the email adress ).

And then rww-play will use the given FOAF instead of creating one.
Optionally, if the  FOAF profile contains a triple about prefered subdomain name, nothing needs to be typed by the user.

The initial subscription page could accept a drag'n'drop for the  FOAF profile : either its URI , or the entire Turtle string.

**Comment by bblfish** [comment]
I agree that we need to consider the use case of a user with an existing WebID who would like to create a new subdomain here.  We could find their certificate and use that. For people with an existing foaf profile but no WebID, then it's a bit difficult to verify their WebID. Currently we are trying to do a bit of verification, mostly so that we can help people in case things go wrong.

As for an ontology of preferred subdomain names, I think that is going too far at present. The gain is too small at present, and it would not be very useful. We're not really thinking of creating preferred subdomains everywhere. Rather we are hoping people will buy their own domain names.

**Comment by slorber** [comment]
I think we should avoid the copy/paste of serialized RDF. It is not really user friendly and will only reach the scientific semantic world

**Comment by jmvanel** [comment]
Not having this feature,
it's like saying 
"OK, we have some RDF inside, but we don't care about the semweb community."

Agreed, the semweb community is not as large as we would wish, but it's sizable.
Why turning the back on it ?

Also , the semantic semantic Web is all about sharing data, so why not import any RDF (Turtle) that people already have.
